doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -170,7 +170,12 @@ Gazell
 Matter
 ------
 
-* Added FastTrack Recertification and Portfolio Certification programs.
+* Added:
+
+  * FastTrack Recertification and Portfolio Certification programs.
+  * Support for configuring the Matter over Thread applications to use OpenThread API directly instead of utilizing Zephyr L2 networking layer.
```

Comment:
```suggestion
  * Matter-over-Thread apps can now use the OpenThread API directly, bypassing the Zephyr L2 layer.
```

Reply:
Applied, but replaced bypassing with `instead of using intermediate Zephyr L2 layer.`, because we are not bypassing it, we just disabled it at all.